# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/ipc/DocAccessibleParent.h
# Commit: d1558d374cef
# Full Hash: d1558d374cef9d954a84b7db4646e48a4878f621
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 1: Provide an argument to PlatformCaretMoveEvent specifying whether the event was caused by user input. r=eeejay
#   
#   This is needed in order to support find in page on Android using remote caret events instead of virtual cursor change events.
#   
#   Depends on D192641
# ==============================================================================

diff -r d6f61c448b90 -r d1558d374cef accessible/ipc/DocAccessibleParent.h
--- a/accessible/ipc/DocAccessibleParent.h	Tue Nov 28 21:50:11 2023 +0000
+++ b/accessible/ipc/DocAccessibleParent.h	Tue Nov 28 22:03:19 2023 +0000
@@ -110,7 +110,8 @@
   mozilla::ipc::IPCResult RecvCaretMoveEvent(
       const uint64_t& aID, const LayoutDeviceIntRect& aCaretRect,
       const int32_t& aOffset, const bool& aIsSelectionCollapsed,
-      const bool& aIsAtEndOfLine, const int32_t& aGranularity) final;
+      const bool& aIsAtEndOfLine, const int32_t& aGranularity,
+      const bool& aFromUser) final;
 
   virtual mozilla::ipc::IPCResult RecvTextChangeEvent(
       const uint64_t& aID, const nsAString& aStr, const int32_t& aStart,